families/settings/sawtooth_settings/src/handler.rs

```diff
@@ -15,7 +15,7 @@
  * ------------------------------------------------------------------------------
  */
 
- cfg_if! {
+cfg_if! {
```

Comment:
Squash this into the commit that introduced it (see other comments).

Reply:
I've made this its own Fix formatting commit now.